hack/update-profile-manifests.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
Is this script specific for this repo of do we have it copied to multiple repos?

Reply:
Hi @dmage so far the thinking is that it would be copied to multiple repos, given that we don't have a good way of sharing scripts across multiple components. It could be placed in library-go, but not all components use library-go. Also not all components use the same directories for their manifests. I'm definitely open to ideas on how to better handle this.

Reply:
@csrwng what about https://github.com/openshift/build-machinery-go ? `cluster-image-registry-operator` uses only one target from build-machinery, so it shouldn't be a problem to add a single target to other repositories.

See https://github.com/openshift/cluster-image-registry-operator/blob/e5877a88fa9d13a7b69075595c021fce5a884333/Makefile#L9-L13

You can add something similar:

```
$(call add-profile-manifests,./manifests/,./profile-patches/) 
```

Reply:
That sounds promising, will try it out. Thx!

Reply:
Submitted https://github.com/openshift/build-machinery-go/pull/26 - will update this PR when that merges